package.json

```diff
@@ -10,7 +10,8 @@
   "dependencies": {
     "core-js": "^3.6.5",
     "vue": "^3.0.0",
-    "vue-router": "^4.0.0-0"
+    "vue-router": "^4.0.0-0",
+    "vueperslides": "^3.2.0"
```

Comment:
Please pull in the changes made to main! This was already done by Sanaa for her section, so this might result in merge conflicts.
Do the follow:
`git rebase main`